components/cluster.py

```diff
@@ -163,7 +163,7 @@ def callback_wrapper(n_clicks, threshold, start_date, end_date):
         return "Select dates"
     print("Computing clusters...")
     start_date = dt.strptime(start_date,  "%Y-%m-%d")
-    end_date = dt.strptime(end_date,  "%Y-%m-%d")
+    end_date = dt.strptime(end_date,  "%Y-%m-%d") + datetime.timedelta(days=1)
```

Comment:
I believe something like this was in one of the first versions of the calendar component :D  the prodigal son comes back home.  :rofl:

Reply:
Oh, really? Must've just got lost on the way :smile: